Formula/avahi.rb

```diff
@@ -0,0 +1,44 @@
+class Avahi < Formula
+  desc "Service Discovery for Linux using mDNS/DNS-SD"
+  homepage "https://avahi.org"
+  url "https://github.com/lathiat/avahi/archive/v0.8.tar.gz"
+  sha256 "c15e750ef7c6df595fb5f2ce10cac0fee2353649600e6919ad08ae8871e4945f"
+  license "LGPL-2.1-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "intltool" => :build
+  depends_on "libtool" => :build
+  depends_on "m4" => :build
+  depends_on "perl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "xmltoman" => :build
+  depends_on "dbus"
+  depends_on "glib"
+  depends_on "libdaemon"
+  depends_on :linux
+
+  def install
+    system "./bootstrap.sh", "--disable-debug",
+                             "--disable-dependency-tracking",
+                             "--disable-silent-rules",
+                             "--prefix=#{prefix}",
+                             "--sysconfdir=#{prefix}/etc",
+                             "--localstatedir=#{prefix}/var",
+                             "--disable-mono",
+                             "--disable-monodoc",
+                             "--disable-python",
+                             "--disable-qt4",
+                             "--disable-qt5",
+                             "--disable-gtk",
+                             "--disable-gtk3",
+                             "--disable-libevent",
+                             "--with-distro=none",
+                             "--with-systemdsystemunitdir=no"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "avahi-browse #{version}", shell_output("#{bin}/avahi-browse --version").chomp
```

Comment:
Nothing else we can test?

Reply:
Agreed, there should be something better.  I'll look for some examples that will work in Docker.

Reply:
I wrote a much more extensive test now that includes some headers and links to several of the shared libraries.